docs/changelog.md

```diff
@@ -7,11 +7,14 @@
 ## Unreleased
 
 - Add support for Python 3.13.
+- Support qiskit >=2.0
```

Comment:
Maybe put in a bit more clear that we are dropping support for qiskit 1.x?

Reply:
Thanks. See [3864d5f](https://github.com/CQCL/pytket-qiskit/pull/475/commits/3864d5f39673df8a4be328eebb2b645956b6a19a).